packages/graph-explorer/src/connector/openCypher/queries/fetchSchema.ts

```diff
@@ -140,30 +149,47 @@ const fetchEdgesAttributes = async (
     return edges;
   }
 
-  await Promise.all(labels.map(async labelResult => {
-    const edgesTemplate = edgesSchemaTemplate({
-      type: labelResult
-    });
-
-    const response = await openCypherFetch<RawEdgesSchemaResponse>(edgesTemplate);
-
-    const edge = response.results[0].object as OCEdge;
-    const label = edge["~entityType"] as string;
-    const properties = edge["~properties"];
-    edges.push({
-      type: label,
-      displayLabel: sanitizeText(label),
-      total: countsByLabel[label],
-      attributes: Object.entries(properties || {}).map(([name, prop]) => {
-        const value = prop;
-        return {
-          name,
-          displayLabel: sanitizeText(name),
-          dataType: typeof value === "string" ? "String" : "Number",
-        };
-      }),
-    });
-  }));
+  await Promise.all(
+    labels.map(async labelResult => {
+      const edgesTemplate = edgesSchemaTemplate({
+        type: labelResult,
+      });
+
+      const response = await openCypherFetch<RawEdgesSchemaResponse>(
```

Comment:
Are we handling exceptions during the fetch?

Reply:
An exception here would cause the entire sync to fail. This is being caught in `useSchemaSync` hook.

https://github.com/aws/graph-explorer/blob/372bb59847db7cdebe4c2b1f84dc1e50ec4ee3bd/packages/graph-explorer/src/hooks/useSchemaSync.ts#L26-L34